packages/components/src/components/assets/Icon/icons.ts

```diff
@@ -38,6 +38,10 @@ export const ICONS = {
     DASHBOARD: require('../../../images/icons/dashboard.svg'),
     DATA: require('../../../images/icons/data.svg'),
     DISCREET: require('../../../images/icons/discreet.svg'),
+    ADDRESS_CONTINUES: require('../../../images/icons/address_continues.svg'),
+    ADDRESS_NEXT: require('../../../images/icons/address_next.svg'),
+    ADDRESS_PIXEL_CONTINUES: require('../../../images/icons/address_pixel_continues.svg'),
```

Comment:
nitpick: `continuous` would be maybe slightly better than `continues`?

Reply:
Hmm I called it `continues` because like the address continues after this icon, not sure it `continuous` is better